src/engine/Sim3D.ts

```diff
@@ -594,14 +601,41 @@ export class Sim3D extends EventEmitter {
   }
 
   private updatePhysics(time: number): void {
-    this.simObjects.forEach((simObject) => {
-      simObject.object.update(time);
-    });
+    /* Timestep is set to constsant to avoid large jumps
+       from wreaking havok in the world.
+
+       Time is split into fixed size chunks (TIME_STEP)
+       the remaining time is saved for the next update
+    */
+
+    // Add remaining time from previous frame
+    time += this.time_delta;
+
+    let steps = Math.floor(time / TIME_STEP);
 
-    // Timestep is set to constsant to avoid large jumps
-    // wreaking havok in the world.
-    this.world.step(1 / 60, 10, 8);
-    this.world.clearForces();
+    // Prevent huge jumps in time
+    if (steps > MAX_TIME_STEPS) {
+      console.warn(
+        "Large jump in time. Skipping simulation steps:",
+        steps,
+        ">",
+        MAX_TIME_STEPS
+      );
+      steps = MAX_TIME_STEPS;
+      time = 0; // We don't need the delta when skipping.
+    }
+
+    // Remaining time delta is stored for later...
+    this.time_delta = time % TIME_STEP;
+
+    for (let i = 0; i < steps; i++) {
+      // Update forces on objects for this time step
+      this.simObjects.forEach((simObject) => {
+        simObject.object.update(TIME_STEP);
+      });
+      this.world.step(TIME_STEP, 10, 8);
+      this.world.clearForces();
```

Comment:
I think this call is not needed actually.  The [world's `step`](https://github.com/shakiba/planck.js/blob/b8c946c/src/dynamics/World.ts#L833) function clears the forces automatically, link to the [function doc](https://github.com/shakiba/planck.js/blob/3f89cb77bc64ebc91d9c3a9795f659a5e3e2ba07/docs/api/classes/world.md#clearforces). Obviously it's not doing any harm either, it's just mildly confusing to a novice.